pages/pil/read/index.js

```diff
@@ -18,6 +19,21 @@ module.exports = settings => {
     next();
   });
 
+  app.get('/', (req, res, next) => {
+    if (!process.env['ENABLE_PIL_REVIEW']) {
+      return next();
+    }
+    const needsReview = moment(req.pil.reviewDate).isBefore(moment().add(3, 'months'));
+    if (needsReview) {
+      res.locals.static.pilReviewRequired = {
+        reviewUrl: req.buildRoute('pil.review'),
+        overdue: moment(req.pil.reviewDate).isBefore(moment())
+      };
+    }
```

Comment:
I think this work might be better done API side.

Eventually we're going to want to make API calls which list/count reviews that are due, and so having the logic all in one place makes more sense.

Reply:
you're not wrong